src/components/Layout/Header/Header.tsx

```diff
@@ -33,6 +35,8 @@ export const Header = () => {
     dispatch,
   } = useWallet()
 
+  const isWalletConnectToDappsAvailable = useFeatureFlag('WalletConnectToDapps')
```

Comment:
Nitpick, inconsistent naming
```suggestion
  const isWalletConnectToDappsEnabled = useFeatureFlag('WalletConnectToDapps')
```